Open Judge System/Workers/OJS.Workers.ExecutionStrategies/PhpProjectExecutionStrategy.cs

```diff
@@ -0,0 +1,98 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using OJS.Common;
+    using OJS.Common.Extensions;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.Executors;
+
+    public class PhpProjectExecutionStrategy : ExecutionStrategy
+    {
+        private const string ZippedSubmissionName = "_$Submission";
+        private const string SuperGlobalsTemplateName = "_Superglobals.php";
+        private const string SubmissionEntryPoint = "index.php";
+        private const string SuperGlobalsRequireStatementTemplate = "<?php require_once '##templateName##'; ?>";
```

Comment:
Why not directly replace "##templateName##" with `SuperGlobalsTemplateName` in interpolated string in a `readonly` field instead of `const` and use just this in `RequireSuperGlobalsTemplateInUserCode` method, without using `.Replace`

Reply:
Extensibility. In the future we might expand the `RequireStatementTemplate` or simply add more/different templates, so using a different variable(s) allows us to easily slot in any new changes. Also it is easier to read IMO.